lib/meta-boxes.php

```diff
@@ -119,6 +126,20 @@ function igv_cmb_metaboxes() {
   ) );
 
   $obra_meta->add_field( array(
+    'name'       => __( 'Image', 'cmb2' ),
+    'desc'       => __( 'if single image not slideshow', 'cmb2' ),
+    'id'         => $prefix . 'image',
+    'type'       => 'file',
+  ) );
```

Comment:
hmm good point. not sure rly. perhaps flexibility is good or too many options confuse the user. what do you think?